index.bs

```diff
@@ -282,30 +266,164 @@ Loader instances are initially created with the internal slots described in the
   <tr>
     <td>\[[Registry]]</td>
     <td>An object</td>
-    <td>An instance of Registry (<a href="#registry">4</a>).</td>
+    <td>An instance of <a href="#registry">Registry</a>.</td>
   </tr>
 </table>
 
 <h2 id="registry">Registry Objects</h2>
 
-<h3 id="registry-abstract-operations">Abstract Operations for Registry Objects</h3>
+<h3 id="registry-constructor">The Registry Constructor</h3>
+
+The Registry constructor is the %Registry% intrinsic object. When called as a constructor it creates and initializes a new %Registry% object. %Registry% is not intended to be called as a function and will throw an exception when called in that manner.
```

Comment:
It's not clear to me whether we need to expose the ability to construct new registry objects as a userland function or not. AFAICT the question comes down to subclassing: do subclasses need the ability to specify their own registry, and therefore need the ability to construct new ones (or new instances of a subclass of Registry)?

If they do, then we probably need to have Loader take an argument specifying the registry object. Note that a consequence of this would be that multiple loaders could end up sharing a registry. (This _might_ be fine, even though I've avoided it in the past; every built-in loader in the browser would by default have a distinct registry but you could choose to share registries explicitly if you wanted to.)

But if none of this is necessary, and it's fine for loaders, whether direct instances of Loader or instances of subclasses, to forcibly and non-overrideably get their \[\[Registry]] field automatically set to a new instance of %Registry%, then it doesn't make much sense to me to expose this constructor functionality.